misc/python/materialize/mzcompose.py

```diff
@@ -854,6 +857,44 @@ def __init__(
         )
 
 
+class Toxiproxy(PythonService):
+    def __init__(
+        self,
+        name: str = "toxiproxy",
+        image: str = "shopify/toxiproxy:2.1.4",
+        port: int = 8474,
+    ) -> None:
+        super().__init__(
+            name=name,
+            config={
+                "image": image,
+                "ports": [port],
+            },
+        )
+
+
+class Localstack(PythonService):
+    def __init__(
+        self,
+        name: str = "localstack",
+        image: str = f"localstack/localstack:0.12.5",
+        port: int = 4566,
+        services: str = "sts,iam,s3,sqs,kinesis",
```

Comment:
If we're going to break out this environment variable we can make it a `List[str]` and join it internally before appending it to the environment

Reply:
Ok, will do, thanks.